conf/app.routes

```diff
@@ -1,264 +1,271 @@
+GET         /:taxYear/:businessId/expenses/change-work-from-business-premises                                  controllers.journeys.expenses.tailoring.WorkFromBusinessPremisesController.onPageLoad(taxYear: Int, businessId: String, mode: Mode = CheckMode)
+POST        /:taxYear/:businessId/expenses/change-work-from-business-premises                                  controllers.journeys.expenses.tailoring.WorkFromBusinessPremisesController.onSubmit(taxYear: Int, businessId: String, mode: Mode = CheckMode)
+
+GET         /:taxYear/:businessId/expenses/travel                                                              controllers.journeys.expenses.tailoring.TravelForWorkController.onPageLoad(taxYear: Int, businessId: String, mode: Mode = NormalMode)
+POST        /:taxYear/:businessId/expenses/travel                                                              controllers.journeys.expenses.tailoring.TravelForWorkController.onSubmit(taxYear: Int, businessId: String, mode: Mode = NormalMode)
+GET         /:taxYear/:businessId/expenses/change-travel                                                       controllers.journeys.expenses.tailoring.TravelForWorkController.onPageLoad(taxYear: Int, businessId: String, mode: Mode = CheckMode)
+POST        /:taxYear/:businessId/expenses/change-travel                                                       controllers.journeys.expenses.tailoring.TravelForWorkController.onSubmit(taxYear: Int, businessId: String, mode: Mode = CheckMode)
+
+GET         /:taxYear/:businessId/expenses/advertising-marketing                                               controllers.journeys.expenses.tailoring.AdvertisingOrMarketingController.onPageLoad(taxYear: Int, businessId: String, mode: Mode = NormalMode)
+POST        /:taxYear/:businessId/expenses/advertising-marketing                                               controllers.journeys.expenses.tailoring.AdvertisingOrMarketingController.onSubmit(taxYear: Int, businessId: String, mode: Mode = NormalMode)
+GET         /:taxYear/:businessId/expenses/change-advertising-marketing                                        controllers.journeys.expenses.tailoring.AdvertisingOrMarketingController.onPageLoad(taxYear: Int, businessId: String, mode: Mode = CheckMode)
+POST        /:taxYear/:businessId/expenses/change-advertising-marketing                                        controllers.journeys.expenses.tailoring.AdvertisingOrMarketingController.onSubmit(taxYear: Int, businessId: String, mode: Mode = CheckMode)
+
+GET         /:taxYear/:businessId/expenses/entertainment                                                       controllers.journeys.expenses.tailoring.EntertainmentCostsController.onPageLoad(taxYear: Int, businessId: String, mode: Mode = NormalMode)
```

Comment:
If you merge main, all taxYears should be of type TaxYear now instead of Int

Reply:
and the single amount journey pages will be moved out of the other expenses subjourneys